src/Tools/dotnet-watch/test/dotnet-watch.Tests.csproj

```diff
@@ -12,6 +12,9 @@
     -->
     <UseAspNetCoreSharedRuntime>true</UseAspNetCoreSharedRuntime>
     <DoNotApplyWorkaroundsToMicrosoftAspNetCoreApp>true</DoNotApplyWorkaroundsToMicrosoftAspNetCoreApp>
+
+    <!-- Skipped due to https://github.com/dotnet/aspnetcore/issues/26061 -->
+    <SkipTests Condition="'$([System.Runtime.InteropServices.RuntimeInformation]::IsOSPlatform($([System.Runtime.InteropServices.OSPlatform]::OSX)))' == 'true'">true</SkipTests>
```

Comment:
Thanks for the FYI - lack of any coverage for `dotnet watch` on OSX is concerning.  Let's make sure we try to investigate ASAP.

Reply:
We have a follow up to this that I think should address it: https://github.com/dotnet/aspnetcore/pull/26069/. Getting this in to resolve the flakiness and I'll undo it as part of the follow up.